inc/classes/class-genesis-featured-widget-amplified.php

```diff
@@ -915,41 +907,37 @@ public function form( $instance ) {
 		foreach ( $columns as $column => $boxes ) {
 			if ( 'col1' === $column ) {
 				echo '<div style="float: left; width: 250px;">';
-
 			} else {
 				echo '<div style="float: right; width: 250px;">';
 			}
 
 			foreach ( $boxes as $box ) {
-
 				echo '<div style="background: #f1f1f1; border: 1px solid #DDD; padding: 10px 10px 0 10px; margin-bottom: 5px;">';
 
 				foreach ( $box as $field_id => $args ) {
-
 					$class = $args['save'] ? 'gfwa-widget-control-save' : '';
 					$style = $args['requires'] ? ' style="' . gfwa_get_display_option( $instance, $args['requires'][0], $args['requires'][1], $args['requires'][2] ) . '"' : '';
 
 					switch ( $args['type'] ) {
-
 						case 'post_type_select':
 							echo '<p><label for="' . esc_attr( $this->get_field_id( $field_id ) ) . '">' . str_replace( esc_html( '<br />' ), '<br />', esc_html( $args['label'] ) ) . ':</label>
-								<select class="' . esc_html( $class ) . '" id="' . esc_attr( $this->get_field_id( $field_id ) ) . '" name="' . esc_attr( $this->get_field_name( $field_id ) ) . '">';
+								<select class="' . esc_html( $class ) . '" id="' . esc_attr( $this->get_field_id( $field_id ) ) . '" name="' . esc_attr( $this->get_field_name( $field_id ) ) . '">'; // WPCS: XSS ok.
 
-							$args       = array(
+							$args                               = array(
 								'public' => true,
 							);
-							$output     = 'names';
-							$operator   = 'and';
-							$post_types = get_post_types( $args, $output, $operator );
-							$post_types = array_filter( $post_types, 'gfwa_exclude_post_types' );
+													$output     = 'names';
+													$operator   = 'and';
+													$post_types = get_post_types( $args, $output, $operator );
+													$post_types = array_filter( $post_types, 'gfwa_exclude_post_types' );
 
 							foreach ( $post_types as $post_type ) {
 								echo '<option style="padding-right:10px;" value="' . esc_attr( $post_type ) . '" ' . selected( esc_attr( $post_type ), $instance['post_type'], false ) . '>' . esc_attr( $post_type ) . '</option>';
 							}
 
-							echo '<option style="padding-right:10px;" value="any" ' . selected( 'any', $instance['post_type'], false ) . '>' . esc_html__( 'any', 'gfwa' ) . '</option>';
+													echo '<option style="padding-right:10px;" value="any" ' . selected( 'any', $instance['post_type'], false ) . '>' . esc_html__( 'any', 'gfwa' ) . '</option>';
```

Comment:
I’m also confused about the spacing here and the following.

Reply:
As above - phpcbf has gone rogue here. Commit to fix coming shortly.